ironfish/src/rpc/routes/faucet/getFunds.ts

```diff
@@ -29,6 +30,14 @@ router.register<typeof GetFundsRequestSchema, GetFundsResponse>(
   `${ApiNamespace.faucet}/getFunds`,
   GetFundsRequestSchema,
   async (request, node): Promise<void> => {
+    // check node network id
+    const networkId = node.internal.get('networkId')
+
+    if (networkId !== DEFAULT_NETWORK_ID) {
```

Comment:
Let's hardcode the testnet network ID here, since we're planning to change `DEFAULT_NETWORK_ID` to mainnet when we launch mainnet.

I think it's okay to hardcode it since right now I think we hardcode the network IDs everywhere. (except for `DEFAULT_NETWORK_ID`). If you want, it could be useful in a separate PR to make an enum of Network IDs, so we can reference `NetworkID.Testnet` rather than just hardcoding `0`.

Reply:
I will hardcode the network id in this PR. I will make the enum of network Id as a follow up when I have time.